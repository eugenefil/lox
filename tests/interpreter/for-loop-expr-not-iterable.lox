for ch in 5 {}
