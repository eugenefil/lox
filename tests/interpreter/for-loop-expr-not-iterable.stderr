error: 'Number' is not iterable
 --> $DIR/for-loop-expr-not-iterable.lox:1:11
  |
1 | for ch in 5 {}
  |           ^
